ix-dev/community/esphome/templates/docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% set c1 = tpl.add_container(values.consts.esphome_container_name, "image") %}
+
+{% do c1.set_user(values.run_as.user, values.run_as.group) %}
+
+{% do c1.healthcheck.set_test("tcp", {"port": values.consts.internal_webui_port}) %}
+
+{% do c1.environment.add_env("ESPHOME_DASHBOARD_USE_PING", true) %}
+{% do c1.environment.add_user_envs(values.esphome.additional_envs) %}
+
+{% do c1.ports.add_port(values.network.web_port, values.consts.internal_webui_port) %}
```

Comment:
Is internal port configurable? 
That would make much easier to allow enabling host network

Reply:
i looked [here ](https://github.com/esphome/esphome/blob/dev/docker/Dockerfile)and it seems hardcoded to 6052. Also found this [feature request](https://github.com/esphome/feature-requests/issues/2738) for the same